web/locales/he.yml

```diff
@@ -1,79 +1,79 @@
 he:
   TextDirection: 'rtl'
-  Dashboard: Dashboard
-  Status: Status
-  Time: Time
-  Namespace: Namespace
-  Realtime: Real-time
-  History: History
-  Busy: Busy
-  Processed: Processed
-  Failed: Failed
-  Scheduled: Scheduled
-  Retries: Retries
-  Enqueued: Enqueued
-  Worker: Worker
-  LivePoll: Live Poll
-  StopPolling: Stop Polling
-  Queue: Queue
-  Class: Class
-  Job: Job
-  Arguments: Arguments
-  Extras: Extras
-  Started: Started
-  ShowAll: Show All
-  CurrentMessagesInQueue: Current jobs in <span class='title'>%{queue}</span>
-  Delete: Delete
-  AddToQueue: Add to queue
-  AreYouSureDeleteJob: Are you sure you want to delete this job?
-  AreYouSureDeleteQueue: Are you sure you want to delete the %{queue} queue?
-  Queues: Queues
-  Size: Size
-  Actions: Actions
-  NextRetry: Next Retry
-  RetryCount: Retry Count
-  RetryNow: Retry Now
-  Kill: Kill
-  LastRetry: Last Retry
-  OriginallyFailed: Originally Failed
-  AreYouSure: Are you sure?
-  DeleteAll: Delete All
-  RetryAll: Retry All
-  NoRetriesFound: No retries were found
-  Error: Error
-  ErrorClass: Error Class
-  ErrorMessage: Error Message
-  ErrorBacktrace: Error Backtrace
-  GoBack: ← Back
-  NoScheduledFound: No scheduled jobs were found
-  When: When
-  ScheduledJobs: Scheduled Jobs
-  idle: idle
-  active: active
-  Version: Version
-  Connections: Connections
-  MemoryUsage: Memory Usage
-  PeakMemoryUsage: Peak Memory Usage
-  Uptime: Uptime (days)
-  OneWeek: 1 week
-  OneMonth: 1 month
-  ThreeMonths: 3 months
-  SixMonths: 6 months
-  Failures: Failures
-  DeadJobs: Dead Jobs
-  NoDeadJobsFound: No dead jobs were found
-  Dead: Dead
-  Processes: Processes
-  Thread: Thread
-  Threads: Threads
-  Jobs: Jobs
-  Paused: Paused
-  Stop: Stop
-  Quiet: Quiet
-  StopAll: Stop All
-  QuietAll: Quiet All
-  PollingInterval: Polling interval
-  Plugins: Plugins
-  NotYetEnqueued: Not yet enqueued
-  CreatedAt: Created At
-  BackToApp: Back to App
+  Dashboard: לוח מחוונים
+  Status: מצב
+  Time: שעה
+  Namespace: מרחב שם
+  Realtime: זמן אמת
+  History: היסטוריה
+  Busy: עסוקים
```

Comment:
`עסוקים` is plural. Is that right for this context? 
If you wanted singular then `עסוק` is the right word.

I should also note that `עסוק` is "masculine" noun and some words in hebrew require masculine while others require feminine so it really depends on *who* is busy? 
If it's a "he" in hebrew then indeed `עסוק` is right, but if it's a "she" in hebrew them it's `עסוקה`.
Lastly, if it's plural feminine then it's `עסוקות`. 
Tough language, I know... 

Anyway, if you told me whether it's *plural or singular* plus *what's the noun `busy` referring to* then I'd tell you what's the correct translation.